src/GeneralStateTestsFiller/VMTests/vmTests/push0Filler.yml

```diff
@@ -0,0 +1,275 @@
+        5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f\
+        5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f\
+        5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f\
+        5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f\
+        5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f\
+        5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f\
+        5f"
+      nonce: '0'
+      storage: {}
+
+    0000000000000000000000000000000000000400:
+      balance: 0
+      # Update already existing storage value
+      # push1(02) push0 sstore push1(00) push1(01) sstore
+      code: ':raw 0x60025f556000600155'
+      nonce: '0'
+      storage: {0: 0x0a, 1: 0x0a}
+
+    0000000000000000000000000000000000000500:
+      balance: 0
+      code: ':yul { sstore(0, staticcall(100000, 0x0000000000000000000000000000000000000600, 0, 0, 0, 0)) sstore(1, 1) returndatacopy(0x1f, 0x0, 0x1) sstore(0x2, mload(0x00)) }'
+      nonce: 0
+      storage: {}
+
+    0000000000000000000000000000000000000600:
+      balance: 0
+      # push1(0xff) push0 mstore8 push1(0x01) push1(0x00) return
+      code: ':raw 0x60ff5f5360016000f3'
+      nonce: 0
+      storage: {}
+
+  transaction:
+    data:
+      - ':label single_push0 :raw 0x0000000000000000000000000000000000000100'
+      - ':label 1024_push0   :raw 0x0000000000000000000000000000000000000200'
+      - ':label 1025_push0   :raw 0x0000000000000000000000000000000000000300'
+      - ':label push0_upd_storage :raw 0x0000000000000000000000000000000000000400'
+      - ':label push0_upd_storage_sc :raw 0x0000000000000000000000000000000000000500'
+    gasLimit:
+      - 700000
+    gasPrice: 10
+    nonce: 0
+    secretKey: 45a915e4d060149eb4365960e6a7a45f334393093061116b197e3240065ff2d8
+    to: 'b94f5374fce5edbc8e2a8697c15331677e6ebf0b'
+    value:
+    - 0
+        
+  expect:
+    - indexes:
+        data: ':label single_push0'
+        gas:  !!int -1
+        value: !!int -1
+      network:
+        - 'London'
+      result:
+        0000000000000000000000000000000000000100:
+          storage: {}
+        b94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+          storage:
+            0x01: 0x01
+    - indexes:
+        data: ':label single_push0'
+        gas:  !!int -1
+        value: !!int -1
+      network:
+        - 'London+3855'
+      result:
+        0000000000000000000000000000000000000100:
+          storage:
+            0x00: 0x01
+        b94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+          storage:
+            0x00: 0x01
```

Comment:
Can you put more comments what we expect here. (In all expect sections of this test) what does sstore equals 1 mean.

Reply:
Added more comments.